game/opengl_text.cc

```diff
@@ -212,11 +135,13 @@ SvgTxtThemeSimple::SvgTxtThemeSimple(fs::path const& themeFile, float factor) :
 	parseTheme(themeFile, m_text, tmp, tmp, tmp, tmp, a);
 }
 
-void SvgTxtThemeSimple::render(std::string _text) {
-	if (!m_opengl_text.get() || m_cache_text != _text) {
-		m_cache_text = _text;
-		m_text.text = _text;
-		m_opengl_text = std::make_unique<OpenGLText>(m_text, m_factor);
+void SvgTxtThemeSimple::render(std::string const& text) {
+	if (!m_opengl_text.get() || m_cache_text != text) {
+		m_cache_text = text;
+
+		auto renderer = TextRenderer();
+
+		m_opengl_text = std::make_unique<OpenGLText>(renderer.render(text, m_text, m_factor));
```

Comment:
Do we need instance level access or can we make the renderer go static? We're now creating TextRenderer objects for every render call which isn't necessairy a good thing i think?

Reply:
I thought about it, but making the functions static seems to me like premature optimiszation. If we need (or want to have)  an alternative renderer, which could be implemented by interface, the static way is counterproductive, That leads me keep is non static. As alternative:
```
static TextRenderer renderer;
```
Then the consumer avoids frequent initialization.

Reply:
Agree to this approach (consumer of the class defining a static renderer)
We wouldn't want the methods itself to be static like you said, that's counterproductive